go/vm/geth/ct.go

```diff
@@ -190,33 +162,28 @@ func (i *callInterceptor) Call(env *geth_vm.EVM, me geth_vm.ContractRef, addr ge
 		kind = vm.StaticCall
 	}
 
-	var vmValue vm.Value
-	value.FillBytes(vmValue[:])
-
 	res, err := i.makeCall(kind, vm.CallParameter{
 		Sender:    vm.Address(me.Address()),
 		Recipient: vm.Address(addr),
-		Value:     vmValue,
+		Value:     vm.Uint256ToValue(value.Clone()),
```

Comment:
The Clone should be unnecessary, right?
```suggestion
		Value:     vm.Uint256ToValue(value),
```
Also in functions below.